aws_rules_guardduty/aws_guardduty_med_sev_findings.yml

```diff
@@ -8,8 +8,9 @@ LogTypes:
 Tags:
   - AWS
 Severity: Medium
+DedupPeriodMinutes: 60
 Description: >
-  A medium-severity GuardDuty finding has been identified, which could indicate a host-compromise.
+  A medium-severity GuardDuty finding has been identified.
```

Comment:
Does this need a 'title' like the others?

Reply:
what do you mean?

Reply:
Nevermind. I had to open the "Reference" more in the browser  to find the "title": In the ones above they were visible in the displayed snippet.